docs/source/gbd2019_models/risk_risk_correlation/maternal_bmi_hgb_birthweight/index.rst

```diff
@@ -0,0 +1,179 @@
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+=============================================================================
+Maternal BMI, Maternal Hemoglobin, and Infant Birthweight Risk Correlation
+=============================================================================
+
+Risk Exposures Overview
+------------------------
+
+Maternal undernourishment, including correlated body mass index (BMI) and anemia exposures, is associated adverse child health outcomes, including infant birthweight.
+
+Risk Exposures in GBD
+-----------------------
+
+Links to documentation for relevant risk exposure pages include:
+
+- :ref:`GBD 2019 Low birthweight short gestation risk exposure <2019_risk_exposure_lbwsg>`
+
+- :ref:`Maternal BMI risk exposure conditional on hemoglobin status <2019_risk_exposure_maternal_bmi_hgb>`
+
+- :ref:`Hemoglobin exposure <2019_hemoglobin_model>`
+
+Vivarium Modeling Strategy
+----------------------------
+
+Correlation
+++++++++++++
+
+.. note::
+
+   The strategy for modeling these risk-risk correlations related was developed for the needs of the :ref:`Intravenous Iron Intervention simulation <2019_concept_model_vivarium_iv_iron>`. Different strategies may be more appropriate for different project needs and should be reevaluated when necessary.
+
+We will model an association between a joint categorical distribution of maternal BMI and hemoglobin with infant birthweight. The asssoication between maternal BMI/hemoglobin categories will be measured in continuous shifts in infant birthweight. The magnitude of these shifts will be informed from BMGF trial data, shown below:
+
+.. note::
+
+   Modeling strategy may be adapted if BMGF data is available for proportion born with low BW (<2500g) only (rather than mean birthweight). Briefly, in this case, we will use the probability born low birth weight for each maternal BMI/hemoglobin category to sample from LBWSG exposure categories with BW<2500 grams or from categories with BW>=2500g rather than apply shifts to birthweight values sampled from the overall LBWSG exposure distribution.
+
+.. list-table:: Maternal BMI/Hemoglobin category unadjusted birthweight shifts
+   :header-rows: 1 
+
+   *  - Category
+      - Maternal BMI exposure
+      - Maternal Hemoglobin exposure
+      - Birthweight shift
+      - Note
+   *  - cat4
+      - >=18.5
+      - >=10
+      - 0
+      - TMREL/Reference group
+   *  - cat3
+      - <18.5
+      - >=10
+      - TBD
+      - Relative to cat4
+   *  - cat2
+      - >=18.5
+      - <10
+      - TBD
+      - Relative to cat4
+   *  - cat1
+      - <18.5
+      - <10
+      - TBD
+      - Relative to cat4
+   
+Given that,
+
+.. math::
+
+   BW_\text{cat{1,2,3} - cat4} = \overline{BW}_\text{cat{1,2,3}} - \overline{BW}_\text{cat4}
+
+.. math::
+
+   \overline{BW}_{GBD} = \sum_{cat}^{n} p_\text{cat} * \overline{BW}_\text{cat}
```

Comment:
Are these lines being calculated with GBD data? I thought GBD did not have mean birth weights by BMI/hemoglobin categories.

My expectation would have been to calculate the shifts for each category relative to the population mean in the BMGF data and then apply the relevant shift to the simulant's birthweight exposure value (pulled from the GBD distribution).

Reply:
Yes, I see your point, thanks for your close review. I changed the "GBD" reference here to "population" to avoid confusion around this point and then will update the specifics when we get the structure of the data from BMGF and see exactly how we can apply it.